app/src/components/blocks/_lloydGeorge/lloydGeorgeSelectSearchResults/LloydGeorgeSelectSearchResults.tsx

```diff
@@ -96,12 +123,17 @@ const AvailableFilesTable = ({
                                     <Checkboxes onChange={handleChangeCheckboxes}>
                                         <Checkboxes.Box
                                             value={result.ID}
+                                            id={result.ID}
                                             data-testid={`checkbox-${index}`}
                                             checked={selectedDocuments.includes(result.ID)}
+                                            aria-checked={selectedDocuments.includes(result.ID)}
                                         >
-                                            <span className="nhsuk-u-visually-hidden">
+                                            <label
+                                                htmlFor={result.fileName}
+                                                className="nhsuk-u-visually-hidden"
+                                            >
                                                 {result.fileName}
-                                            </span>
+                                            </label>
```

Comment:
If we look into the html this renders, the original Checkboxes.Box already create a <label> tag for us. Adding a <label> here will create a nested tag with an incorrect for= attribute, which probably affect screenreader functionality.
I would suggest to revert to previous version
![image](https://github.com/user-attachments/assets/2de2d4f6-a318-4b59-a480-e30ce553fbd3)

Reply:
Done